gnd/build.gradle

```diff
@@ -128,10 +134,10 @@ dependencies {
     implementation "com.jakewharton.rxbinding2:rxbinding-design:$project.rxBindingVersion"
 
     // Android Arch
-    implementation "android.arch.lifecycle:runtime:$project.lifecycleVersion"
-    implementation "android.arch.lifecycle:extensions:$project.lifecycleVersion"
-    implementation "android.arch.lifecycle:reactivestreams:$project.lifecycleVersion"
-    annotationProcessor "android.arch.lifecycle:compiler:$project.lifecycleVersion"
+    implementation "androidx.lifecycle:lifecycle-runtime:2.0.0-beta01"
```

Comment:
How about a variable (like lifecycleVersion) for the androidx.lifecycle version? That will make it easier to keep these consistent.